code/app/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("com.diffplug.spotless")
+}
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        target("src/*/java/**/*.kt")
+        ktlint("0.41.0")
```

Comment:
Use this [constant](https://github.com/adobe/aepsdk-commons/blob/dev/android/aepsdk-gradle-plugin/aepsdk-gradle-plugin/src/main/kotlin/com/adobe/marketing/mobile/gradle/BuildConstants.kt#L39) instead of hardcoded version